pyClarion/base/components.py

```diff
@@ -258,35 +266,48 @@ def features(self) -> Tuple[feature, ...]:
         return self._features
 
     def update(self) -> None:
-        """Set domain properties."""
+        """
+        Set domain properties.
+        Also reflect changes in domain to all relevant places
+        """
 
         pass
 
     @contextmanager
-    def config(self):
-        """Update self after adjustments to config."""
+    def config(self): 
```

Comment:
The docstring should be a bit more precise: the adjustments are not to `_config` itself, but to monitored attributes, as defined by `_config`.